### DISCUSSION TITLE
Not working on Firefox version 121.0

### DISCUSSION THREAD

**Discussion by afraidofwomen28** [xiaoxiaoflood/firefox-scripts#295] [General Support]
I have used this before and absolutely loved it but awhile back it stopped working and I never attempted to reinstall it. I am trying to reinstall it but even as I follow the installation steps to a T, I do not see the button and no scripts I add to the installation work at all. I am on Firefox Version 121.0, and don't know if that has anything to do with this not working. But any help would be appreciated!! :)

**Comment by 117649** [comment]
This repo has been actually set aside since 115 time. Ever since there is only scattered fixes made by individual user. You have to go through issues area to find the newest working fix people have shared. Sadly until somebody took over the responsibility of maintenance it will keep this way.

**Reply by afraidofwomen28** [reply]
Very sad :( Thank you for the quick reply though
